audio-worklet/test/bitcrusher-test.js

```diff
@@ -0,0 +1,129 @@
+/**
```

Comment:
You have 3 different test tasks in a big global scope. These tests need to be separated into isolated scopes and should be orthogonal to each other. Please consider refactoring.